posthog/models/plugin.py

```diff
@@ -206,6 +213,8 @@ class PluginConfig(models.Model):
     # - e.g: "undefined is not a function on index.js line 23"
     # - error = { message: "Exception in processEvent()", time: "iso-string", ...meta }
     error: models.JSONField = models.JSONField(default=None, null=True)
+    # Used to access web.ts from a public URL
```

Comment:
I probably missed something but I don't really get what's this web token for. Can you expand on this comment?

Reply:
Web apps will be injected from `/web_js/:id/:token/`. I don't want somebody to `id++` in a loop and download all possible apps.

Sidenote: this definitely should go behind some CDN/cache, but in v0.2.

Reply:
For me the primary distinction between token and ID is:
- a token serves authentication purposes and can change throughout its entity's life
- an ID serves unique identification purposes and cannot change

I don't see how authentication would work for a `posthog-js` feature, so currently I'd say the only real job of this code is identifying the app to fetch code for. And if the only reason for not using `id` is preventing enumeration, could a plugin UUID (UUIDT) – which would be more general than `web_token` – work here?

Reply:
Good points! Generally agree, though we might want cache invalidation as well. Tokens due to their refreshability serve well for this purpose.